docs/integrations/aws-org/discovery.md

```diff
@@ -0,0 +1,101 @@
+---
+id: discovery
+title: AWS Organization Discovery
+sidebar_label: Discovery
+# prettier-ignore
+description: Automatically discovering AWS accounts from AWS organizations
+---
+
+:::info
+This documentation is written for [the new backend system](../../backend-system/index.md) which is the default since Backstage [version 1.24](../../releases/v1.24.0.md).
+:::
+
+## AWS Organization Provider
+
+The AWS Organization integration has a discovery provider for ingesting AWS accounts from an AWS organization into the software catalog. It will list organization accounts (including the org account) and provide entities to your software catalog. The current provider provides accounts as kind `Resource` with no owners. There are plans to make the kind and ownership customizable soon to allow for flexible requirements.
+
+AWS recommends [organizing your AWS environment using multiple accounts](https://docs.aws.amazon.com/whitepapers/latest/organizing-your-aws-environment/organizing-your-aws-environment.html). You may have dozens or even hundreds of AWS accounts. This provider will help you manage ownership and the lifecycle of your accounts in Backstage.
+
+## Installation
+
+You will have to add the AWS Organization Entity provider to your backend as it is not installed by default, therefore you have to add a dependency on `@backstage/plugin-catalog-backend-module-aws-org` to your backend package.
+
+```bash
+# From your Backstage root directory
+yarn --cwd packages/backend add @backstage/plugin-catalog-backend-module-aws-org
+```
+
+And then update your backend by adding the following line:
+
+```ts title="packages/backend/src/index.ts"
+backend.add(import('@backstage/plugin-catalog-backend'));
+/* highlight-add-start */
+backend.add(import('@backstage/plugin-catalog-backend-module-aws-org'));
+```
+
+## Configuration
+
+To use the discovery provider, you'll need an AWS Organization and AWS credentials (prefferably an IAM role) your Backstage instance can assume.
+
+Create an IAM role in your organization account with the following permissions. Adjust the trust relationship to match your environment. For example, if you run Backstage in ECS fargate allow your task role to assume a role in your organization account to list accounts.
+
+```json
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "VisualEditor0",
+      "Effect": "Allow",
+      "Action": [
+        "organizations:ListAccounts",
+        "organizations:ListTagsForResource"
+      ],
+      "Resource": "*"
+    }
+  ]
+}
+```
+
+Once your role is provisioned add the required backstage AWS configuration. More information on this Backstage integration can be found [here](https://github.com/backstage/backstage/tree/master/packages/integration-aws-node#readme).
+
+This example shows an IAM role in the organization account that can be assumed by backstage (running in AWS).
+
+```yaml
+integrations:
```

Comment:
Every time I've used the AWS auth its been configured with `aws` at the root, not under `integrations`. The TechDocs documentation reflects this:

https://backstage.io/docs/features/techdocs/using-cloud-storage/#configuring-aws-s3-bucket-with-techdocs

This should probably be consistent, and I'm not sure if this is just how it was documented here or the code forces this.

Reply:
Ah, I’m not sure why I had it under integrations. I actually realized I need to go back and test the use of the AWS integration. I was testing this in my org account and it was working but not as I intended. I will test and get this updated!